components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml

```diff
@@ -421,6 +421,68 @@ data:
           "title": "TaskRun Controller kickoff after Kubernetes scheduling checks pass",
           "type": "stat"
         },
+        {
+          "description": "The number of validated ResolutionRequests not yet processed by the Resolver Controller.",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "thresholds"
+              },
+              "mappings": [],
+              "noValue": "No data",
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 30
+                  }
+                ]
+              },
+              "unit": "percent"
```

Comment:
congrats on your first infra-deployment merge @divyansh42 !!

that said, I noticed this minor error only after merge, and not during my review / testing of your change

rather than percent, this line ^^ should be 

`"unit": "none"`

craft a quick PR when you come in on Friday and I'll tag for merge when I get in.

Reply:
PR https://github.com/redhat-appstudio/infra-deployments/pull/4354 is up